### ISSUE TITLE
Not an Issue, A Question

### ISSUE THREAD

**Issue by spencerthayer** [sudo97/Roku-IPTV#3]
Is this the source for the black M3U player? Or is this just a rendered version prepped for sideloading onto a Roku? In other words, can the community start developing with this code alone or is there missing data?

**Comment by sudo97** [comment]
black M3U was forked once from this project. I write some improvements when I see it is good and interesting, but not quite often, only when I know exactly what to write. sideload it if you like, it will work well. do you want to develop? I would be glad to see your pull request and investigate it.

**Comment by spencerthayer** [comment]
Yes, I am interested in developing on it. However I cannot sideload the application because I am being told the manifest isn't correct.

**Comment by spencerthayer** [comment]
`Install Failure: No manifest. Invalid package.`

**Comment by sudo97** [comment]
First of all make sure you have your manifest in the root of the archive:
`unzip -l YourApp.zip | grep manifest`
and the second: if it doesn't work anymore, write your own, you're big boy now, I suppose :-)

**Comment by spencerthayer** [comment]
LOL

**Comment by ghost** [comment]
Back on github.  Just forked over the project again and will try to merge the current m3ublack player code I have.  Appears I went to a different scenegraph template.  Not sure how this merging will work back to the original fork.

**Comment by sudo97** [comment]
always remember: "Entities are not to be multiplied without necessity"

**Comment by sudo97** [comment]
Also, this is not that hard to merge layout, I appreciate your job anyway

**Comment by ghost** [comment]
Keeping/merging the previous many changes made - "I had to rewrite many things, but he made good job." Might have it in a few days.

**Comment by aarongranatto** [comment]
> 
> 
> `Install Failure: No manifest. Invalid package.`

you have to unzip the folder, select all the files inside it and then make a new .zip without the folder (only the files inside it)
